### PR TITLE
Added oc adm drain options for 3.9

### PR DESCRIPTION
**Pull request by ahardin-rh**
Added per discussion in https://github.com/openshift/openshift-docs/pull/5709#issuecomment-367043544

@pravisankar PTAL

### CODE REVIEW

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -241,7 +241,7 @@ controller] can be evacuated; the replication controllers create new pods on
 other nodes and remove the existing pods from the specified node(s). Bare pods,
 meaning those not backed by a replication controller, are unaffected by default.
 
-To evacuate one or more nodes:
+To evacuate all or selected pods on one or more nodes:
 
 ----
 $ oc adm drain <node1> <node2>
```
Comment (pravisankar) [inline admin_guide/manage_nodes.adoc]: We are saying 'evacuate all or selected pods' but we are not telling how to do that using a command.
We should talk about `--pod-selector` here and show that as optional param to the command.
```
 $ oc adm drain <node1> <node2> [--pod-selector=<pod_selector>]
```
I've assumed '[ ]' is used for optional flags if that is not the case use what ever convention is used in other places.

File: admin_guide/manage_nodes.adoc [hunk 2]
```diff
@@ -285,6 +285,19 @@ is drained):
...
+
+----
+$ oc adm drain <node1> <node2>  --dry-run=true
+----
+
+Instead of specifying specific node names (for example, `<node1> <node2>`), you
+can use the `--selector=<node_selector>` option to evacuate pods on selected
+nodes.
+
+You can also use `--pod-selector=<pod>` to filter pods on the node.
```
Comment (pravisankar) [inline admin_guide/manage_nodes.adoc]: This can be skipped if we have already mentioned this option before.

**Review by pravisankar — CHANGES_REQUESTED** [review]

**Comment by ahardin-rh** [comment]
@pravisankar Thanks! I made some edits. Please let me know if more changes are necessary. Thanks again!

**Review by pravisankar — APPROVED** [review]
LGTM

**Pull request merged**